types.go

```diff
@@ -126,6 +126,31 @@ type ItemSection struct {
 	Title string `json:"title"`
 }
 
+// Controls the auto-fill behavior of a website.
+//
+// For more information, visit https://support.1password.com/autofill-behavior/
+type AutofillBehavior string
+
+const (
+	// Auto-fill any page that’s part of the website, including subdomains
+	AutofillBehaviorAnywhereOnWebsite AutofillBehavior = "AnywhereOnWebsite"
+	// Auto-fill only if the domain (hostname and port) is an exact match.
+	AutofillBehaviorExactDomain AutofillBehavior = "ExactDomain"
+	// Never auto-fill on this website
+	AutofillBehaviorNever AutofillBehavior = "Never"
+)
+
+type Website struct {
+	// The website URL
+	URL string `json:"url"`
+	// The label of the website, e.g. 'website', 'sign-in address'
+	Label string `json:"label"`
+	// The auto-fill behavior of the website
+	//
+	// For more information, visit https://support.1password.com/autofill-behavior/
+	AutofillBehavior AutofillBehavior `json:"autofill_behavior"`
```

Comment:
The snake case yaml translation because we forgot the rename camel case should be fixed before releasing